dockers/docker-orchagent/orchagent.sh

```diff
@@ -3,6 +3,7 @@
 # Export platform information. Required to be able to write
 # vendor specific code.
 export platform=`sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type`
+export device=`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`
```

Comment:
I think the variable naming here is confusing. `device` is actually what we commonly refer to as `platform` elsewhere and `platform` is what we commonly refer to as `asic_type` (or preferably, `asic_vendor`). I would prefer it if we stick to a common naming convention.

Reply:
It is a mess because platform is being asic_type in the first place. There are codes everywhere in swss using platform.

Reply:
Ugh. That's unfortunate, because I'm really not a fan of this convention; it's confusing.